app/shares.php

```diff
@@ -30,13 +30,6 @@
 		'form' => 'advanced',
 		'method' => 'GET',
 	),
-	'blogotext' => array(
-		'url' => '~URL~/admin/links.php?url=~LINK~',
-		'transform' => array(),
-		'help' => 'http://lehollandaisvolant.net/blogotext/fr/',
```

Comment:
I don't know anything about blogotext, but I will make the semantic point that unmaintained for a little while isn't the same thing as broken. This help link is broken though; presumably that should be something like https://github.com/BlogoText/blogotext instead.

So I'm on the side of keep. It's not like something with fixes for PHP 7.3/8.0 is something hugely outdated that'll only run on 5+ year old systems, even if it might end up there a few years down the line.

Reply:
feel free to refuse this PR.

feel free to fix the link.

As I feel free to delete old stuff that does not make sense to maintain. Blogotext creates 2 PR where blogotext code had to change (1 PR that reordered the array, 1 PR that will fix the link). I do not like technical debts.

If we do not want to delete it I would suggest to mark blogotext as deprecated with the hint that users of FreshRSS and blogotext should sent a message to this issue to mark it as "still in use".

Reply:
One of the issues is that we have, by design, no telemetry for which features are used, and very little feedback from our user-base, so I tend to be conservative and wait a bit more before deleting things. I can see some user activity from April this year on https://github.com/BlogoText/blogotext/issues
Having a deprecated indication could be interesting.